mtools/mlaunch/mlaunch.py

```diff
@@ -56,6 +56,9 @@ class MongoConnection(Connection):
     """
 
     def __init__(self, *args, **kwargs):
+        if pymongo_version[0] >= 4:
+            if 'directConnection' not in kwargs:
+                kwargs['directConnection'] = True
```

Comment:
Since the pymongo version is always >=3.12 we can change this method to:
```python
        kwargs.setdefault('directConnection', True)
        kwargs.setdefault('serverSelectionTimeoutMS', 1)
        # Set client application name for MongoDB 3.4+ servers
        ...
```